bug_report_page.php

```diff
@@ -230,16 +230,21 @@
 layout_page_begin( __FILE__ );
 
 $t_form_encoding = '';
+$t_form_class = '';
 if( $t_show_attachments ) {
 	$t_form_encoding = 'enctype="multipart/form-data"';
+	$t_form_class = 'class="dropzone-form"';
 }
 ?>
 <div class="col-md-12 col-xs-12">
 <form id="report_bug_form"
 	method="post" <?php echo $t_form_encoding; ?>
 	action="bug_report.php?posted=1"
-	class="dropzone-form"
-	<?php print_dropzone_form_data() ?>>
+	<?php echo $t_form_class ?>
```

Comment:
Is there a special reason that you did not use the same approach you used in https://github.com/mantisbt/mantisbt/commit/5898638f65f08fdadce35864a3c4011dc0fac866 ?

No additional variable $t_form_class, but `echo ' class="dz dropzone-form" ';` just if upload is enabled.